clients/term/client_term.c

```diff
@@ -434,6 +435,14 @@ void editorProcessKeypress() {
 	}
 }
 
+void SiganlHandler(int SignalCaught)
+{
+	if(SignalCaught == SIGWINCH)
```

Comment:
1. Use 4 spaces for indentation, instead of  tabs.
2. Space before parenthesis - `if (..)`